tests/install_upgrade_operators/crypto_policy/test_hco_override_api_server_crypto_policy.py

```diff
@@ -51,7 +51,6 @@ def updated_apiserver_with_tls_old_profile(
 
 @pytest.mark.polarion("CNV-9368")
 def test_hco_overriding_apiserver_crypto_policy(
```

Comment:
@hmeir while I don't want skip fixtures, you can always use jira marker to skip tests. https://github.com/rhevm-qe-automation/pytest_jira/blob/master/README.rst
```
@pytest.mark.jira("RHSTOR-6566", run=False)
```
Would skip the test as long as the jira is not resolved. Just an alternative. 1 hour timeout makes me a bit uncomfortable.